modules/features/manatime_feature_timesheets/manatime_feature_timesheets.module

```diff
@@ -43,10 +104,69 @@ function manatime_feature_timesheets_cron() {
       $timesheet->field_date[LANGUAGE_NONE][0]['value'] = $beginning->format('Y-m-d H:i:s');
       $timesheet->field_date[LANGUAGE_NONE][0]['value2'] = gmdate('Y-m-d H:i:s', $end);
       $timesheet->field_user[LANGUAGE_NONE][0]['target_id'] = $uid;
+      $timesheet->status = 0;
       node_object_prepare($timesheet);
       node_save($timesheet);
     }
     // Set the varible to the next time the function must execute itself.
     variable_set('manatime_feature_timesheets_execution_time', $next_execution_time);
   }
+
+  // Set the time the timesheet reminder will be executed.
+  // Get the variable wich has the time the reminder message will be sended.
+  $manatime_timesheet_reminder = variable_get('manatime_timesheet_reminder', array());
+  $current_day = format_date(REQUEST_TIME, 'custom', $format = 'D');
+  $current_hour = format_date(REQUEST_TIME, 'custom', $format = 'H');
+
+  // Evaluates if it is time to execute the function
+  // manatime_feature_timesheets_create_timesheet_reminder.
+  if ($current_day == $manatime_timesheet_reminder['day'] &&
+  $current_hour <= $manatime_timesheet_reminder['hour'] + 1) {
+    manatime_feature_timesheets_create_timesheet_reminder();
+  }
+}
+
+/**
+ * Send an email to each of the users who have not submitted the timesheet.
+ */
+function manatime_feature_timesheets_create_timesheet_reminder() {
+
+  // Conditional to get only the timesheets of the current week.
+  if (date('w') == 1) {
+    $beginning_date = new DateTime('now');
+  }
+  else {
+    $beginning_date = new DateTime('last monday');
```

Comment:
Remember there's a possible situation when timesheets don't start on monday: if a timesheet has been previously submitted in that week.  Please change code to take this situatio into account.